README.md

```diff
@@ -6,6 +6,14 @@
 
 This will provide you with a starting Drupal project that is managed with Composer. The install will include a small set of contrib modules, a starting custom module for specific for the build, and a custom starting theme generated from Emulsify.
 
+# Installation
+
+## Requirements
+Without these you will have difficulty installing this project.
+
+1. [PHP 7.4](http://www.php.net/)
```

Comment:
@mikeethedude I agree with this, so no change necessary but flagging that 7.4 is technically security upgrades only now: https://www.php.net/supported-versions.php

Reply:
Good to note. We'll want to update everyone soon here. 8.x here we go.